floris/tools/floris_interface.py

```diff
@@ -869,109 +857,25 @@ def get_farm_AEP(
-                at high wind speeds. Defaults to None.
-            yaw_angles (NDArrayFloat | list[float] | None, optional):
-                The relative turbine yaw angles in degrees. If None is
-                specified, will assume that the turbine yaw angles are all
-                zero degrees for all conditions. Defaults to None.
-            turbine_weights (NDArrayFloat | list[float] | None, optional):
-                weighing terms that allow the user to emphasize power at
-                particular turbines and/or completely ignore the power
-                from other turbines. This is useful when, for example, you are
-                modeling multiple wind farms in a single floris object. If you
-                only want to calculate the power production for one of those
-                farms and include the wake effects of the neighboring farms,
-                you can set the turbine_weights for the neighboring farms'
-                turbines to 0.0. The array of turbine powers from floris
-                is multiplied with this array in the calculation of the
-                objective function. If None, this  is an array with all values
-                1.0 and with shape equal to (n_wind_directions, n_wind_speeds,
-                n_turbines). Defaults to None.
-            no_wake: (bool, optional): When *True* updates the turbine
-                quantities without calculating the wake or adding the wake to
-                the flow field. This can be useful when quantifying the loss
-                in AEP due to wakes. Defaults to *False*.
-
-        Returns:
-            float:
-                The Annual Energy Production (AEP) for the wind farm in
-                watt-hours.
-        """
-
-        # Hold the starting values of wind speed and direction
-        wind_speeds = np.array(self.floris.flow_field.wind_speeds, copy=True)
-        wind_directions = np.array(self.floris.flow_field.wind_directions, copy=True)
-
-        # Now set FLORIS wind speed and wind direction
-        # over to those values in the wind rose class
-        wind_speeds_wind_rose = wind_rose.df.ws.unique()
-        wind_directions_wind_rose = wind_rose.df.wd.unique()
-        self.reinitialize(
-            wind_speeds=wind_speeds_wind_rose,
-            wind_directions=wind_directions_wind_rose
-        )
-
-        # Build the frequency matrix from wind rose
-        freq = wind_rose.df.set_index(['wd','ws']).unstack().values
-
-        # Now compute aep
-        aep = self.get_farm_AEP(
-            freq,
-            cut_in_wind_speed=cut_in_wind_speed,
-            cut_out_wind_speed=cut_out_wind_speed,
-            yaw_angles=yaw_angles,
-            turbine_weights=turbine_weights,
-            no_wake=no_wake)
-
-
-        # Reset the FLORIS object to the original wind speed and directions
-        self.reinitialize(
-            wind_speeds=wind_speeds,
-            wind_directions=wind_directions
-        )
```

Comment:
@paulf81 is this removed from here in order to be reimplemented in (or aligned with) the WindRose overhaul?

Reply:
Yes, that was the idea